tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -0,0 +1,258 @@
+import csv
+import datetime
+import json
+import zoneinfo
+
+import pytest
+
+from sds_data_manager.lambda_images.instruments.batch_starter import (
+    all_dependency_present,
+    get_filename_from_event,
+    get_process_details,
+    prepare_data,
+    query_dependencies,
+    query_dependents,
+    remove_ingested,
+)
+
+
+@pytest.fixture(scope="session")
+def mock_event():
+    """Example of the type of event that will be passed to
+    the instrument lambda (in our case batch_starter.py).
+    """
+    with open("../test-data/codicehi_event.json") as file:
+        event = json.load(file)
+    return event
+
+
+@pytest.fixture()
+def setup_database(postgresql):
+    """Populate test database."""
+
+    cursor = postgresql.cursor()
+    cursor.execute("CREATE SCHEMA IF NOT EXISTS sdc;")
+
+    # Drop the table if it exists, to start with a fresh table
+    cursor.execute("DROP TABLE IF EXISTS sdc.codicehi;")
+
+    sql_command = """
+    CREATE TABLE sdc.codicehi (
+        -- Primary key
+        id SERIAL PRIMARY KEY,
+
+        -- Basic columns
+        filename TEXT UNIQUE NOT NULL,
+        instrument TEXT NOT NULL,
+        version INTEGER NOT NULL,
+        level TEXT NOT NULL,
+        mode TEXT,
+        date TIMESTAMP WITH TIME ZONE DEFAULT NULL,
+        ingested TIMESTAMP WITH TIME ZONE DEFAULT (now() AT TIME ZONE 'UTC'),
+        mag_id INTEGER,
+        spice_id INTEGER,
+        parent_codicehi_id INTEGER,
+        status TEXT
+    );
+    """
+
+    cursor.execute(sql_command)
+
+    # Insert mock data:
+    with open("../test-data/process_kickoff_test_data.csv") as f:
```

Comment:
This seems like a lot of formatting work below. Is there a way you can just hardcode the rows inside here to make it more obvious what is going on, or store the stripped values directly in the csv.